jerry-core/ecma/builtin-objects/ecma-builtin-arraybuffer.c

```diff
@@ -0,0 +1,108 @@
+/* Copyright 2016 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "ecma-alloc.h"
+#include "ecma-builtins.h"
+#include "ecma-conversion.h"
+#include "ecma-exceptions.h"
+#include "ecma-gc.h"
+#include "ecma-globals.h"
+#include "ecma-helpers.h"
+#include "ecma-objects.h"
+#include "ecma-arraybuffer-object.h"
+#include "ecma-try-catch-macro.h"
+#include "jrt.h"
+
+#ifndef CONFIG_DISABLE_ARRAYBUFFER_BUILTIN
+
+
+#define ECMA_BUILTINS_INTERNAL
+#include "ecma-builtins-internal.h"
+
+#define BUILTIN_INC_HEADER_NAME "ecma-builtin-arraybuffer.inc.h"
+#define BUILTIN_UNDERSCORED_ID arraybuffer
+#include "ecma-builtin-internal-routines-template.inc.h"
+
+/** \addtogroup ecma ECMA
+ * @{
+ *
+ * \addtogroup ecmabuiltins
+ * @{
+ *
+ * \addtogroup string ECMA ArrayBuffer object built-in
+ * @{
+ */
+
+/**
+ * The ArrayBuffer object's 'isView' routine
+ *
+ * See also:
+ *         ES2015 24.1.3.1
+ *
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value.
+ */
+static ecma_value_t
+ecma_builtin_arraybuffer_object_is_view (ecma_value_t this_arg, /**< 'this' argument */
+                                  ecma_value_t arg) /**< argument 1 */
+{
+  JERRY_UNUSED (this_arg);
+
+  if (ecma_is_value_object (arg))
+  {
+    /* TODO: if arg has [[ViewArrayBuffer]], return true */
+  }
```

Comment:
what is "dead code" here?
And the reason I  didnt implement isView is that  isView need to check typedarray and dataview object